packages/cli-kit/src/public/node/node-package-manager.ts

```diff
@@ -444,7 +453,7 @@ export async function addNPMDependenciesWithoutVersionIfNeeded(
  * @param type - The dependency type.
  * @returns An array with the arguments.
  */
-function argumentsToAddDependenciesWithNPM(dependencies: string[], type: DependencyType): string[] {
+function argumentsToAddDependenciesWithNPM(dependencies: string[], type: DependencyType, exact = false): string[] {
```

Comment:
Having a final boolean parameter is asking for trouble later on. Maybe this should be `{exact} = {exact: false}`?

Otherwise we break the interface if we need to add another param.

Currently it's not exported, so not the end of the world... but it could easily happen without us paying too much attention. Especially in a `public` file.